### DISCUSSION TITLE
Scaling of Rx data

### DISCUSSION THREAD

**Discussion by LUMC-LowFieldMRI** [schote/nexus-console#60] [Q&A] [answered]
What are the units of the receive data? This will be important to see if more/less gain is required on the preamps

**Comment by schote** [comment, marked as answer]
Before the decimation step the receive data is in mV.
This is the unprocessed sinusoidal signal from a signal generator (200 mVpp, 0 mV offset, 2 MHz frequency).
The x-axis shows the number of samples (20 MSps).
![unprocessed_200mVpp_0mV-offset](https://github.com/schote/spectrum-console/assets/42237900/e4a30488-5b47-46cf-bef2-bc5a127b6b65)
